roles/press_common/tasks/main.yml

```diff
@@ -0,0 +1,168 @@
+---
+# Installs Press
+
+# +++
+# Prerequisites
+# +++
+
+- name: install system level dependencies
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
```

Comment:
build tools? can't whatever be built once elsewhere and a resulting package installed instead?

Reply:
This is existing logic that has been moved. I'm not opposed to discussing that at another time though. 😸